{{cookiecutter.repo_slug}}/docs/CONTRIBUTING.md

```diff
@@ -22,30 +22,25 @@ To set up a development environment, please follow these steps:
 
 ## Issues and feature requests
 
-You've found a bug in the source code, a mistake in the documentation or maybe you'd like a new feature? You can help us by submitting an issue to our [GitHub Repository](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/issues). Before you create an issue, make sure you search the archive, maybe your question was already answered.
-{% if cookiecutter.use_github_discussions == 'y' -%}
-Also please check out [GitHub discussions](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/discussions) before submitting an issue. 
-{% endif %}
+You've found a bug in the source code, a mistake in the documentation or maybe you'd like a new feature?{% if cookiecutter.use_github_discussions == 'y' -%} Take a look at [GitHub Discussions](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/discussions) to see if it's already being discussed. {% endif %} You can help us by [submitting an issue on GitHub](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/issues). Before you create an issue, make sure to search the issue archive -- your issue may have already been addressed!
```

Comment:
If we want people to check Discussions first, we should put that first if enabled, and then explain how to submit an issue. Also simplification of some of the language.